### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by igoradamenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,41 @@
...
+2. Ensure `smppex` is started before your application:
 
-## LICENSE
+   ```elixir
+   def application do
+     [applications: [:smppex]]
+   end
+   ```
 
-This software is licensed under [MIT License](LICENSE).
```
Comment (pmenhart) [inline README.md:47]: What is reason to remove mention of the license terms from the readme?

Comment (igoradamenko) [reply README.md:47]: There's license file in the repo: https://github.com/funbox/smppex/blob/master/LICENSE.

I've removed the line just to make the README clear. I may put it back if you find it useful there.

Comment (pmenhart) [reply README.md:47]: When browsing repositories, I find it convenient to learn about license in the same place that describes purpose of the library.
But that's only my preference, others may have different opinion.

Comment (igoradamenko) [reply README.md:47]: Well, if someone finds it useful, why not to put it back. I've reverted it.

Comment (pmenhart) [reply README.md:47]: Thanks. Have a nice day!
Peter

**Comment by savonarola** [comment]
Thanks!

**Pull request merged**